test_redirects/map_globalconf.py

```diff
@@ -322,12 +322,14 @@
     # Bug 920212
     url_test('/firefox/fx/', '/firefox/new/'),
 
-    # Bug 979670, 979531, 1003727, 979664, 979654, 979660
+    # Bug 979670, 979531, 1003727, 979664, 979654, 979660, 1150713
     url_test('/firefox/features/', '/firefox/desktop/'),
     url_test('/firefox/customize/', '/firefox/desktop/customize/'),
     url_test('/firefox/{performance,happy,speed,memory}/', '/firefox/desktop/fast/'),
     url_test('/firefox/security/', '/firefox/desktop/trust/'),
     url_test('/firefox/technology/', 'https://developer.mozilla.org/docs/Tools'),
+    url_test('/firefox/sms/', '/firefox/products/'),
```

Comment:
NIT: Can (if you want) consolidate these two tests into one:

```python
url_test('/firefox/sms/{,sent}', '/firefox/products/'),
```

Not a requirement though. Just FYI.